### ISSUE TITLE
use of B_list['o'] 

### ISSUE THREAD

**Issue by jaypatravali** [seoungwugoh/STM#36]
`            B_list['o'].append( (torch.sum(masks[:,1:o], dim=1) + \
                torch.sum(masks[:,o+1:num_objects+1], dim=1)).clamp(0,1) )`
at https://github.com/seoungwugoh/STM/blob/905f11492a6692dd0d0fa395881a8ec09b211a36/model.py#L217

sorry, I follow this operation can you explain?

**Comment by leeqiaogithub** [comment]
I'm confused, too

**Comment by sugar-fly** [comment]
I have same question,

> ` B_list['o'].append( (torch.sum(masks[:,1:o], dim=1) + \ torch.sum(masks[:,o+1:num_objects+1], dim=1)).clamp(0,1) )` at
> 
> https://github.com/seoungwugoh/STM/blob/905f11492a6692dd0d0fa395881a8ec09b211a36/model.py#L217
> 
> sorry, I follow this operation can you explain?

I have same question, do you have solved it now?

**Comment by seoungwugoh** [comment]
This is a combined mask for other objects. This is only used for multi object case. Otherwise it is zero.
